### ISSUE TITLE
Error when launching nyxt

### ISSUE THREAD

**Issue by deb75** [atlas-engineer/nyxt#1046]
**Describe the bug**

I have just compiled nyxt from the git source. When running the executable I get : 
```
Nyxt version 2-pre-release-4-73-g9ab2d9ce
<INFO> [23:04:34] Listening to socket "/home/user/.local/share/nyxt/nyxt.socket".
<ERROR> [23:04:34] In #<CLOSURE (LAMBDA (&OPTIONAL NYXT::URLS)
                                  :IN
                                  MAKE-STARTUP-FUNCTION) {1007474B6B}>: unknown type specifier: BROWSER
```
and nothing shows up

**Precise recipe to reproduce the issue**
-   git clone https://github.com/atlas-engineer/nyxt.git
-   cd nyxt
-   make  all
- ./nyxt

**Information**
- OS name+version: debian 10
- Graphics card and driver: intel
- Desktop environment / Window manager name+version: awesome wm
- How you installed Nyxt (Guix pack, package manager, build from source): from git
- Information from command copy-system-information:

  If you can't run copy-system-information, please provide the following:
  - Nyxt version (from =M-x nyxt-version= or =nyxt --version=):
  - Lisp implementation/version (if built from source): SBCL 1.4.16.debian
  - Kernel name+version: 5.9.1
  - WebKitGTK+ version: libwebkit2gtk-4.0-37

**Comment by Ambrevar** [comment]
Your SBCL is too old.

You need 1.5 at least.  (Exact version to be confirmed.)

You can install a more modern SBCL with Guix, Roswell, or manually.

We have a Makefile rule that check the SBCL version, funny it didn't
trigger an error for you.  Something may be broken, I'll test again.

**Comment by deb75** [comment]
Hi,

Thanks for your answeer.

Actually, I indeed figured out that the default `sbcl` debian 10 version might be too old by viewing the one used by, e.g., arch.

Upgrading `sbcl` to its the newest version solved the problem.

Regards

**Comment by jmercouris** [comment]
Thanks for the follow-up. Enjoy Nyxt :-)

**Comment by Ambrevar** [comment]
> We have a Makefile rule that check the SBCL version, funny it didn't
> trigger an error for you.  Something may be broken, I'll test again.

It was not stopping the build indeed.

Fixed in 18c20400499b89695e3da0c29c69c080117bb59f.
